cypress/integration/e2e/article.metaAndOphan.e2e.spec.js

```diff
@@ -0,0 +1,118 @@
+import { getPolyfill } from '../../lib/polyfill';
+import { disableCMP } from '../../lib/disableCMP.js';
+import { setLocalBaseUrl } from '../../lib/setLocalBaseUrl.js';
+
+describe('E2E Page rendering', function () {
+	before(getPolyfill);
+
+	beforeEach(function () {
+		disableCMP();
+		setLocalBaseUrl();
+	});
+
+	const getExists = (selector) => cy.get(selector).should('have.length', 1);
+
+	describe('for WEB the document renders with the correct meta and analytics info', function () {
+		it(`Has a head element`, function () {
+			cy.visit(
+				`/Article?url=https://www.theguardian.com/lifeandstyle/2021/jan/21/never-conduct-any-business-naked-how-to-work-from-bed-without-getting-sacked`,
+			);
+			// cy.get(`head`).should('have.length', 1);
+			getExists(`head`);
+		});
+		it(`Required meta SEO fields exist`, function () {
+			const metaTags = [
+				{
+					name: 'description',
+					value:
+						'From warming up your voice to avoiding spillages, here are some tips for keeping up professional appearances',
+				},
+				{
+					name: 'viewport',
+					value: 'width=device-width,minimum-scale=1,initial-scale=1',
+				},
+			];
+
+			metaTags.map((meta) =>
```

Comment:
Do we need the map? You could just have two sets of assertions?

Reply:
IMO, I think it's clearer that *all* the values of the array do the *exact* same thing - that way it's easier to parse quickly and understand a single test than if they were separate and you had to carefully parse each assertion

Reply:
That's true. One point though is that people will only ever land here when something fails and they will likely be on a how can I fix this fast ride so the less context they need to absorb the best. Also, if you need to fix a failing test, you don't want to have to check the remaining tests that are sharing that code.

I'm probably overthinking it though, these cases are pretty simple, it's more a general principle thing which sounds a lot like I'm being dogmatic!

Reply:
And to be clear, I think in this case - the array map - I'm actuially in favour of the way it is here!

Reply:
Yeah, definitely a good general principle, I think this is an odd case where there's no user interaction, just a 'did we render everything ok' where this is probably fairly clear (I had a failed cypress test in Github on this PR and it was quick to understand 😄 )